slp_drawio/resources/schemas/drawio_mapping_schema.json

```diff
@@ -0,0 +1,47 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "title": "Mapping File",
+  "type": "object",
+  "required": ["trustzones", "components", "dataflows"],
+  "properties": {
+    "trustzones": {
+      "type": "array",
+      "items": {
+        "type": "object",
+        "required": ["label"],
+        "properties": {
+          "label": {"$ref": "#/definitions/query"},
+          "type": {"$ref": "#/definitions/query"}
+        }
+      }
+    },
+    "components": {
+      "type": "array",
+      "items": {
+        "type": "object",
+        "required": ["label"],
+        "properties": {
+          "label": {"$ref": "#/definitions/query"},
+          "type": {"$ref": "#/definitions/query"}
+        }
+      }
+    },
+    "dataflows": {
```

Comment:
In diagrams we won't map dataflow, IMHO this value should be removed and keep it the simpler.
It is already removed in DEV for slp_visio processor